subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -141,9 +153,8 @@ static void smp_conn_params_enable(struct bt_conn *conn)
 {
 	struct conn_param_data *cpd = get_conn_param_data(conn);
 
-	if (cpd->state & SMP_CONN_PARAMS_ENABLED) {
-		cpd->state |= SMP_CONN_PARAMS_REQUIRED;
-	} else {
+	cpd->state |= SMP_CONN_PARAMS_REQUIRED;
```

Comment:
Discussed offline.
The `SMP_CONN_PARAMS_REQUIRED` may be set even if the actual request fails to be sent (conn param update returns an error). If there would be no more SMP requests for some time, then we may spuriously send SMP conn param request on next connection parameter update initiated by peer. That would happen no matter what was the time between the mentioned last SMP request and peer-initiated conn param update.

Reply:
Still, there would be no long-lasting consequences of this behavior. The default connection parameters would be restored by the dedicated work after timeout.